boards/arm/da1469x_dk_pro/da1469x_dk_pro.dts

```diff
@@ -72,3 +74,10 @@
 &gpio1 {
 	status = "okay";
 };
+
+&uart {
+	status = "okay";
+	current-speed = <115200>;
+	tx-pin = <9>;
+	rx-pin = <8>;
```

Comment:
this is no longer allowed, please, create a pinctrl driver, ref. https://docs.zephyrproject.org/latest/hardware/pinctrl/index.html